src/lib.rs

```diff
@@ -94,3 +94,23 @@ pub extern "C" fn __swift_bridge__null_pointer() -> *const std::ffi::c_void {
 
 #[doc(hidden)]
 pub fn assert_copy<T: Copy>() {}
+
+#[no_mangle]
+#[doc(hidden)]
+pub extern "C" fn __swift_bridge__equality_operator_for_RustStr(
+    lhs: string::RustStr,
+    rhs: string::RustStr,
+) -> bool {
+    if lhs.len != rhs.len {
```

Comment:
Let's make this `lhs == rhs`

And then you can

```rust
impl PartialEq for RustStr { .. }
```

And handle the comparison inside of the `PartialEq` impl.